dockerfiles/alpine/Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY bob-backend/Cargo.toml bob-backend/Cargo.toml
 COPY bob-common/Cargo.toml bob-common/Cargo.toml
 COPY bob-grpc/Cargo.toml bob-grpc/Cargo.toml
 COPY bob-apps/Cargo.toml bob-apps/Cargo.toml
+RUN sed -i "s|[[bench]]|[[bench_ignore]]|g" */Cargo.toml
```

Comment:
Build error: `failed to parse manifest at '/usr/src/bob/bob/Cargo.toml'`.
`sed` uses regex syntax, thus `[` and `]` must be escaped with a backslash